include/glow/Partitioner/Partitioner.h

```diff
@@ -157,7 +158,9 @@ class Partitioner {
   Partitioner(Module *parent, const std::vector<DeviceInfo> &devices);
 
   /// Decompose each function in a module and return a list of DAGNodes.
```

Comment:
Probably should update the comment since this no longer returns the list of DAGNodes.